src/components/MaanedsbeloepAvansertBeregning/Felles/PensjonVisningDesktop.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+
+import { BodyLong, Box, HStack, VStack } from '@navikt/ds-react'
+
+import { formatUttaksalder } from '@/utils/alder'
+
+import { Pensjonsdata } from '../hooks'
+import { PensjonDataVisning } from './PensjonDataVisning'
+
+import styles from '../MaanedsbeloepAvansertBeregning.module.scss'
+
+interface Props {
+  pensjonsdata: Pensjonsdata[]
+  summerYtelser: (data: Pensjonsdata) => number
+  hentUttaksmaanedOgAar: (alder: Alder) => { maaned: string; aar: string }
+}
+
+export const PensjonVisningDesktop: React.FC<Props> = ({
+  pensjonsdata,
+  summerYtelser,
+  hentUttaksmaanedOgAar,
+}) => {
+  const intl = useIntl()
+
+  if (!pensjonsdata.length) return null
+
+  return (
+    <HStack gap="8" width="100%" className={styles.maanedsbeloepDesktopOnly}>
+      {pensjonsdata.map((data, index) => (
+        <Box
+          key={`desktop-${index}`}
+          marginBlock="1 0"
+          borderRadius="medium"
+          paddingInline="6"
+          paddingBlock="4"
+          background="bg-subtle"
+          maxWidth={{ sm: '27rem', md: '31rem' }}
+          flexGrow="1"
+        >
+          <VStack gap="1">
+            <BodyLong
+              size="medium"
+              weight="semibold"
+              data-testid="maanedsbeloep-desktop-title"
+            >
+              {intl.formatMessage({
+                id: 'beregning.avansert.maanedsbeloep.box_title',
+              })}
+              {formatUttaksalder(intl, data.alder)}{' '}
+              {data.alderspensjon &&
+                !data.afp &&
+                !data.pensjonsavtale &&
+                `(${hentUttaksmaanedOgAar(data.alder).maaned} ${hentUttaksmaanedOgAar(data.alder).aar})`}
```

Comment:
Teksten blir "stykket opp" i HTML-en, noe som ikke er ideelt for skjermesere:
![bilde](https://github.com/user-attachments/assets/cdb91d73-6a39-4d1c-8103-b5419bf34122)
(Gjør blant annet at skjermleseren tar en pause etter "Ved")

Kan for eksempel løses ved å putte teksten i en template literal:
```suggestion
              {`${intl.formatMessage({
                id: 'beregning.avansert.maanedsbeloep.box_title',
              })}
              ${formatUttaksalder(intl, data.alder)}`}
              {data.alderspensjon &&
                !data.afp &&
                !data.pensjonsavtale &&
                ` (${hentUttaksmaanedOgAar(data.alder).maaned} ${hentUttaksmaanedOgAar(data.alder).aar})`}
```

Reply:
Eller du kan bruke `+` som jeg ser du har gjort noen andre steder 😄